### DISCUSSION TITLE
HTTPS OF MY REPOSITORY

### DISCUSSION THREAD

**Discussion by learnRight1** [community/community#62412] [Repositories] [answered]
### Select Topic Area

Question

### Body

I'm not seeing the https link after creating a new repository on my github account, I made the repo private and checked the readme before clicking the create button, it disappear. I can still see the repo among my repository but the https link is not coming up when I click the code tab. Please, kindly help to resolve this, I have refreshed the page multiple times. thanks.

**Comment by skiks** [comment]
First, check if your repository is still set to private. Go to the repository's page, click on "Settings," and make sure it's set as "Private."

Double-check that you're logged in with the right GitHub account, and you actually own the repository or have the right permissions. If not, ask the owner to add you as a collaborator.

Sometimes, browser issues can mess things up. So, try clearing your browser cache or even use a different browser to see if that helps.

Make sure you have a stable internet connection, as a shaky one can cause issues with loading the repository info.

You might wanna take a look at the GitHub Status page to see if there are any problems with their service right now.

Try opening the repository page in an incognito or private browsing window. That way, we can check if any browser extensions or settings are messing things up.

If nothing works, you can reach out to GitHub Support for help. Go to their support page and let them know what's happening. The more details you can give them, the better they can assist you.

To contact GitHub Support, you can visit their support page: https://support.github.com/contact

**Comment by learnRight1** [comment]
I really appreciate the prompt response and I got it solved using another
browser.Thanks, I'm really grateful. Should I change the browser that I'll
now be using for it going forward or what?
Thanks in anticipation again.

**Reply by skiks** [reply, marked as answer]
You're welcome! I'm glad to hear that using another browser helped you solve the issue with your GitHub repository.

As for whether you should change the browser you'll be using for GitHub going forward, it's entirely up to you. If the new browser is working well for you and you haven't experienced any other issues, you can continue using it for GitHub without any problems.

However, if you prefer your previous browser or have specific reasons for using it, you can try a few things before making a decision:

Clear Browser Cache: Sometimes, clearing the cache of your original browser can resolve the problem. It might have been a temporary glitch, and after clearing the cache, GitHub may work fine on your preferred browser as well.

Update Browser: Ensure that your original browser is up to date. Outdated versions can sometimes cause compatibility issues with certain websites.

Check Browser Extensions: If you have any browser extensions installed, try disabling them temporarily to see if any of them were causing the problem. Certain extensions can interfere with the functionality of websites.

Test GitHub Again: After trying the above steps, test GitHub on your original browser once more to see if the issue persists.

Btw, what browser do you use?

**Comment by learnRight1** [comment]
Thanks once again, I cannot thank you enough for the help and prompt
response. I'm indeed grateful. I'll surely do all that you have suggested
in regards to the browser cache.
Thanks and have a great day.

**Reply by skiks** [reply]
I would greatly appreciate it if you consider marking my answer as the best solution for this discussion

**Comment by learnRight1** [comment]
Why not? How can I possibly do that. Step by step guide on how to give my
comment.

**Reply by skiks** [reply]
Below the comment, click Mark as answer.

**Comment by learnRight1** [comment]
Please I'll be glad to do but I've been looking for where I logged the
complaint and I'm finding it difficult to trace my path back there again.
Can you help with step by step guide on how to do that.
Thanks in anticipation.
